app/assets/javascripts/3dbio_viewer/src/webapp/components/protvista/Protvista.module.css

```diff
@@ -1,8 +1,30 @@
+body {
+  font-family: Lato, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+
 .title {
     font-weight: bold;
-    margin-bottom: 5px;
+    margin-bottom: 15px;
+    color: #123546;
 }
 
 .actions {
     float: right;
 }
+
+button {
```

Comment:
Same for button, this will change CSS for all buttons in the page, so it makes more sense on App.css